config/environment.rb

```diff
@@ -17,7 +17,8 @@
   'federicogalassi',
   'timvisher',
   'udioica',
-  'braxler'
+  'braxler',
+  'Sakigw'
```

Comment:
I don't believe it should be a problem, but for consistency can we lowercase?

Reply:
If it is not a problem then sure, but I don't know about that. The URL http://www.vimgolf.com/Sakigw shows the user page while http://www.vimgolf.com/sakigw does not.